deepchecks/nlp/utils/text_properties.py

```diff
@@ -870,44 +900,47 @@ def calculate_builtin_properties(
     )
     import_warnings = set()
 
-    progress_bar = create_progress_bar(
-        iterable=list(raw_text),
-        name='Text Samples Calculation',
-        unit='Text Sample'
-    )
-    for text in progress_bar:
-        progress_bar.set_postfix(
-            {'Sample': truncate_string(text, max_length=20) if text else 'EMPTY STRING'},
-            refresh=False
-        )
-        if pd.isna(text):
+    with tqdm(total=int(len(raw_text)//batch_size)) as pbar:
+        for i in range(0, len(raw_text), batch_size):
```

Comment:
you can do tqdm(range(0, len(raw_text), batch_size)) than you dont need to manually update pbar